processors/base/writers/publication.py

```diff
@@ -43,7 +43,7 @@ def write_publication(conn, publication, source_id):
         create = True
 
     # Write object only for high priority source
-    if create or source_id in ['pubmed']:
+    if create or source_id in ['pubmed', 'hra']:
```

Comment:
@vitorbaptista @roll 
Why?
*(I added `'hra'` in that list, but feels like a hack that we might not need anymore)*